scripts/config.py

```diff
@@ -52,12 +54,15 @@ def __init__(self):
             openai.api_type = "azure"
             openai.api_base = self.openai_api_base
             openai.api_version = self.openai_api_version
+        else:
+            self.openai_deployment_id = None
 
         self.elevenlabs_api_key = os.getenv("ELEVENLABS_API_KEY")
 
         self.use_mac_os_tts = False
         self.use_mac_os_tts = os.getenv("USE_MAC_OS_TTS")
-        
```

Comment:
This whitespace change is unrelated.

Reply:
@nponeccop, consider using black for formatting/linting. could make life easier with so many PRs to review.